ui/app/components/app/transaction-list-item/transaction-list-item.container.js

```diff
@@ -45,12 +45,12 @@ const mapStateToProps = (state, ownProps) => {
     hasEnoughCancelGas,
     rpcPrefs,
     isDeposit,
+    blockTime: getBasicGasEstimateBlockTime(state),
   }
 }
 
 const mapDispatchToProps = dispatch => {
   return {
-    fetchBasicGasAndTimeEstimates: () => dispatch(fetchBasicGasAndTimeEstimates()),
```

Comment:
This is still used in the `handleRetry` method of the `transaction-list-item` component.